plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeTableProperties.java

```diff
@@ -43,7 +43,7 @@
     //       to pass the column names to ConnectorMetadata#getStatisticsCollectionMetadata; we should consider passing
     //       ConnectorTableHandle instead of ConnectorTableMetadata as an argument since it makes more information
     //       available (including the names of the columns to be analyzed)
-    public static final String ANALYZE_COLUMNS_PROPERTY = "$sep.analyze_columns";
+    public static final String ANALYZE_COLUMNS_PROPERTY = "$trino.analyze_columns";
```

Comment:
Any impact?

Reply:
From looking at the code the this property is used only during the `ANALYZE` command. So I assume that it should have no impact.

@alexjo2144 please do also check this one to be make sure that I'm not mistaken.

Reply:
It should be fine, it's a transient property key that shouldn't be stored in HMS